addons/maptools/functions/fnc_openMapGpsUpdate.sqf

```diff
@@ -1,30 +1,34 @@
 #include "script_component.hpp"
 /*
- * Author: esteldunedain
- * update gps display
- *
- * Arguments:
- * Something
- *
- * Return Value:
- * None
- *
- * Example:
- * call ACE_maptools_fnc_openMapGpsUpdate
- *
- * Public: No
- */
+* Author: esteldunedain, PabstMirror
+* update gps display, called from main map's draw
+*
+* Arguments:
+* 0: Map display <DISPLAY>
+*
+* Return Value:
+* None
+*
+* Example:
+* [findDisplay 12] call ACE_maptools_fnc_openMapGpsUpdate
+*
+* Public: No
+*/
 
-if ((!("ItemGPS" in assigneditems ACE_player)) || {isNull (uiNamespace getVariable [QGVAR(ui_mapGpsDisplay), displayNull])}) exitWith {
-    ("RscACE_MapGps" call BIS_fnc_rscLayer) cutText ["","PLAIN"];  // Close GPS RSC
-    [(_this select 1)] call CBA_fnc_removePerFrameHandler;  // Remove frameHandler
+params ["_mapCtrl"];
+private _mapDisplay = ctrlParent _mapCtrl;
```

Comment:
Function header says "map display" and example uses a DISPLAY, but `ctrlParent` only accepts CONTROL.